delta/json-schema/delta-protocol.ess

```diff
@@ -0,0 +1,842 @@
+namespace Lion.WebCore.Serialization
```

Comment:
As discussed with Pedro, we don't want to merge this file in the end. It's just more convenient to discuss the structures in this format that raw JSONSchema.